### PR TITLE
Add endpoint parameter

### PR DESCRIPTION
**Pull request by cre8**
Add endpoint to use other systems than AWS

### CODE REVIEW

**Comment by SpBills** [comment]
Any reason this hasn't been merged?

File: action.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ inputs:
     required: false
     default: /
     description: 'destination directory for upload'
+  destination_dir:
```
Comment (bluek1te) [inline action.yml:24]: This should be labelled endpoint

Comment (cre8) [reply action.yml]: @bluek1te correct, patched it

**Comment by sgreene570** [comment]
friendly ping @shallwefootball  :) 

This feature would be awesome!

File: index.js [hunk 2]
```diff
@@ -24,7 +28,8 @@ const DESTINATION_DIR = core.getInput('destination_dir', {
 
 const s3 = new S3({
   accessKeyId: AWS_KEY_ID,
-  secretAccessKey: SECRET_ACCESS_KEY
+  secretAccessKey: SECRET_ACCESS_KEY,
+  endpoint: ENDPOINT,
```
Comment (shallwefootball) [inline index.js:32]: this is not optional

**Comment by shallwefootball** [comment]
thx 👍 
https://github.com/shallwefootball/upload-s3-action/releases/tag/v1.3.0

**Pull request closed without merging**